rocAL/rocAL_pybind/examples/rocAL_api_caffe_reader.py

```diff
@@ -71,39 +71,37 @@ def main():
     print("num_classes:: ", num_classes)
     # Create Pipeline instance
     pipe = Pipeline(batch_size=batch_size, num_threads=num_threads, device_id=args.local_rank,
-                    seed=random_seed, rocal_cpu=_rocal_cpu)
+                    seed=random_seed, rocal_cpu=rocal_cpu, output_memory_type = types.CPU_MEMORY if rocal_cpu else types.GPU_MEMORY)
     # Use pipeline instance to make calls to reader, decoder & augmentation's
     with pipe:
         if _rocal_bbox:
-            jpegs, labels, bboxes = fn.readers.caffe(path=image_path, bbox=_rocal_bbox, random_shuffle=True)
-            crop_begin, crop_size, bboxes, labels = fn.random_bbox_crop(bboxes, labels, device="cpu",
-                                    aspect_ratio=[0.5, 2.0],
-                                    thresholds=[0, 0.1, 0.3, 0.5, 0.7, 0.9],
-                                    scaling=[0.3, 1.0],
-                                    ltrb=True,
-                                    allow_no_crop=True,
-                                    num_attempts=1)
-            images = fn.decoders.image_slice(jpegs, crop_begin, crop_size, output_type = types.RGB, path=image_path, annotations_file="", random_shuffle=True,shard_id=local_rank, num_shards=world_size)
+            jpegs, labels, bboxes = fn.readers.caffe(path=image_path, bbox=_rocal_bbox)
+            # crop_begin, crop_size, bboxes, labels = fn.random_bbox_crop(jpegs,# labels, device="cpu",
+            #                         aspect_ratio=[0.5, 2.0],
+            #                         # thresholds=[0, 0.1, 0.3, 0.5, 0.7, 0.9],
+            #                         scaling=[0.3, 1.0],
+            #                         # ltrb=True,
+            #                         allow_no_crop=True,
+            #                         num_attempts=1)
+            images = fn.decoders.image(jpegs, path=image_path, shard_id=local_rank, random_shuffle=True)
```

Comment:
We need to use image_slice when using the random_bbox_crop. Make sure to re-introduce it after adding

Reply:
If we can get random bbox crop working before issuing this PR, I'll reintroduce that and test this example

Reply:
So image_slice and random_bbox_crop is not fixed yet ?